engine/swarm/swarm_manager_locking.md

```diff
@@ -0,0 +1,153 @@
+---
+description: Automatically lock Swarm managers to protect encryption keys
+keywords:
+- swarm, manager, lock, unlock, autolock, encryption
+title: Lock your swarm to protect its encryption key
+---
+
+In Docker 1.13 and higher, the RAFT logs used by swarm managers are encrypted on
+disk by default. This at-rest encryption protects your service's configuration
+and data from attackers who gain access to the encrypted RAFT logs. One of the
+reasons this feature was introduced was in support of the new [Docker
+secrets](secrets.md) feature.
+
+When Docker restarts, the encryption key needed to encrypt and decrypt RAFT logs
+is loaded into each manager node's running memory from disk. Docker 1.13
+introduces the ability to protect the encryption key at rest by _autolocking_
+your swarm.
+
+When Docker restarts or a new manager attempts to join the swarm, or whenever
+the encryption key needs to be loaded from disk to a node's memory, you must
+[unlock the swarm](swarm_manager_locking.md#unlock-a-swarm) first, using a key
+provided by you or generated by Docker when the swarm was locked. You can rotate
```

Comment:
Users cannot provide your own key - it's similar to join tokens.  It is generated by docker when you enable auto-locking, or when you rotate the unlock key.